zcash_client_sqlite/src/lib.rs

```diff
@@ -2198,13 +2260,16 @@ mod tests {
         );
 
         let seed = Secret::new(vec![0u8; 32]);
+        let zip32_index_0 = zip32::AccountId::ZERO;
         let seed_based = st.wallet_mut().create_account(&seed, &birthday).unwrap();
         let seed_based_account = st.wallet().get_account(seed_based.0).unwrap().unwrap();
         let ufvk = seed_based_account.ufvk().unwrap();
 
         assert_matches!(
-            st.wallet_mut().import_account_ufvk(ufvk, &birthday, AccountPurpose::Spending),
+            st.wallet_mut().import_account_hd(&seed, zip32_index_0, &birthday),
```

Comment:
The `import_account_ufvk` check is done in `check_collisions` below, so this is adding an extra check for collisions between `create_account` and `import_account_hd`.